### DISCUSSION TITLE
What's the Return Key #?

### DISCUSSION THREAD

**Discussion by AdrienneELS** [rsjaffe/MIDI2LR#770] [Q&A]
Hi Everyone! I'm struggling to figure out the assignment for the return key. I've searched the web for the key number but nothing is working. Can anyone help? Thanks!

**Comment by rsjaffe** [comment]
Do you mean for **Keyboard Shortcuts**? It is `return`. See https://github.com/rsjaffe/MIDI2LR/wiki/Plugin-Options-Dialog#keyboard-shortcuts

**Reply by AdrienneELS** [reply]
Hi again! @rsjaffe  I can't figure this out for the life of me. I've tried to create a shortcut for return and the assigned button isn't responding. Basically when I'm in crop mode I want to assign a return button to exit crop mode. Also, when I click the right arrow on the Behringer mini, I have to hit it twice to move to the next image, yet the left arrow works perfectly for next image previous. Thanks!

**Comment by antzpantz** [comment]
It's odd that the "return" button isn't working. Have you tried assigning
"return" to other buttons?

Also, you might not necessarily need to have a return button. Just switch
modes with another key/button?

**Reply by AdrienneELS** [reply]
@antzpantz , maybe I'm calling the action of the button something else? For instance, Closing crop mode would bring it to what view? I don't know what it's called, rather I just do it without thinking on a regular keyboard. I've looked the shortcuts up and down and don't see it. I haven't tried assigning to another button. I'll give that a shot!

**Comment by antzpantz** [comment]
I find having a 'return' button redundant when you already have it on the
keyboard. I feel that having a controller should be more like grouping your
most used functions or key combinations onto buttons for your muscle
memory. rather than acting like an extra keyboard.

Generally, once you do your 'crop' or other function, switching to another
mode/function would save that crop unless you press Escape or Cancel.

**Reply by AdrienneELS** [reply]
The problem I'm having is the muscle memory that's already existing lol! I've been using the Xtouch mini for years to run through thousands of photos at a time and find that removing my hand from the xtouch to reach for the keyboard poses a reduction in my workflow speed. At that point, I should use one or the other but I don't want to give up using the xtouch because it was fast and more ergonomic for me. I'm just trying to make this thing work lol. I was using a program before called pfixer that no longer operates or updates with the frequently changing lightroom and photoshop. It was perfect, but now I'm stuck. maybe I'm doing something wrong with shortcut command. Do you mind if I make a few screenshots to show you what I'm doing?

**Comment by antzpantz** [comment]
Sure thing. Post what you're trying to do.

**Comment by FedeCang** [comment]
Hi guys, did you find any solution? I have the same problem. I set up a macro and at end of it I need to press enter to finish the sequence of keys, so without the "return" it wouldn't work.

**Reply by FedeCang** [reply]
<img width="1250" alt="1" src="https://github.com/rsjaffe/MIDI2LR/assets/161979153/d073196a-ada4-4396-8f24-a8c296d76ce7">
<img width="1260" alt="2" src="https://github.com/rsjaffe/MIDI2LR/assets/161979153/31db5d29-01fb-4da7-89fe-8fbbff68de0c">
<img width="849" alt="3" src="https://github.com/rsjaffe/MIDI2LR/assets/161979153/89ffc3eb-72c1-4d04-9d98-b4e744fb5270">

**Comment by heyjeff** [comment]
Capitalize the last Command in that line.
